src/Model/ModelDataCustomised.php

```diff
@@ -49,17 +49,22 @@ public function __isset(string $property): bool
         return isset($this->customised->$property) || isset($this->original->$property) || parent::__isset($property);
     }
 
+    public function forTemplate(): string
+    {
+        return $this->original->forTemplate();
+    }
```

Comment:
Since `forTemplate()` is now implemented in `ModelData` by default, I had to pick one of the original or customised to call here. I chose the original.

Reply:
It's `ModelDataCustomised` but that's usually for customising the _data_, not the _template_.
I'll add a comment to that effect.